aqua/yabai/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        koekeishiya yabai 3.3.0 v
+revision            0
+
+categories          aqua
+platforms           darwin
+license             MIT
+maintainers         nomaintainer
+
+description         Tiling window management for the Mac
+
+long_description    yabai is a window management utility that is designed \
+                    to work as an extension to the built-in window manager \
+                    of macOS
+
+checksums           rmd160  73367ba51d055becbf98c97cf5f8ab95d9a9d1b7 \
+                    sha256  4c7306f77d35a09bba3b228bbe8778c1673e0d6c30c486e9d7bc34169deefcdf \
+                    size    1509115
+
+patchfiles          patch-use-cc-variable.diff
+
+use_configure       no
+
+configure.compiler  clang
```

Comment:
Don't force the compiler to be clang. Let MacPorts choose the right compiler for you. If the port is not compatible with some compilers, you can blacklist them, but shouldn't do so unless you have evidence that a particular compiler is actually incompatible.